CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+## 1.0.1
+
+**May 6, 2024**
+- Set the temp dir to `output_dir` in `download_from_gcs_unpack`.
```

Comment:
It's a bugfix, we need to add information about the bug that was fixed, not just the changes applied